Main.java

```diff
@@ -1,19 +1,40 @@
 class Main {
   public static void main(String[] args) {
-   // kindly use appropriate data types for your declaration
-   //declare an array variable and assign the days of the month debtor defaulted,
-   
-   // declare variables of amount the debtor is to pay per day, and amount the debtor paid and assign values to them
-   
-   // declare all the variables needed for your calculations
+  byte [] days= {1,3,9,19,30};
+  byte amountPerDay=100;
+  float paid=500;
+  float amountToPay=0f;
+  float totalAmount=0f;
+  float extraAmount;
+  float AmountLeftToPay;
+  byte daysPaid;
+  byte daysNotPaid;
+  byte i=0;
 
-   // calculate and print total amount the debtor is to pay using for each loop
-
-   // calculate the days the debtor paid for and print the days in which was paid for starting from the left to the right of the array. Also check if there was a day that the debtor did not pay the complete amount for that day, if yes, print the day and the amount left to pay for that day. 
-
-   // calculate amount the debtor is left to pay if he did not pay complete fees and the days not paid for or  check if the debtor paid extra and the extra amount he paid for, or if he paid the exact amount needed to be paid and print likewise
-
-   //kindly remove the statement below when you are done with the assignment
-    System.out.println("Hello world!");
+  for(byte x : days)
+  totalAmount+=amountPerDay;
+  System.out.println("Total amount to pay is "+totalAmount);
+  daysPaid=(byte)(paid/amountPerDay);
+  System.out.println("Days debtors paid for ");
+  for(i=0;i<5 &&i<daysPaid;i++){
+    System.out.println(days[i]+",");
+  }
+  System.out.println("");
+  if (paid%amountPerDay>0&&i<5){
+    amountToPay=paid%amountPerDay;
+    System.out.println("You did not complete defaultedDays[i]");
+  }
```

Comment:
This is wrong. It should be System.out.println("You did not complete defaultedDays[daysPaid]");